dev/com.ibm.ws.rest.handler.validator/src/com/ibm/wsspi/validator/Validator.java

```diff
@@ -24,16 +24,24 @@
  * Validator implementations should be registered as OSGi service in the service registry.
  */
 public interface Validator {
-    // TODO add custom login module/properties
+
+    /**
+     * Key used to obtain a String representation of the JSON request body.
+     * If present, it will added to the <code>Map&ltString,Object></code> passed into
+     * the <code>validate</code> method.
+     */
+    public static final String JSON_BODY_KEY = "com.ibm.wsspi.validator.jsonBody";
```

Comment:
This is just an implementation detail of the SPI. The end user won't specify any value here, they will just make a JSON the request body. The Validator framework will read this JSON and then add it to the map. I chose a fully-qualified property name here so that we don't conflict with any potential params that a Validator implementation may choose.